packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -70,11 +72,13 @@ export async function createSummarizerFromFactory(
     dataStoreFactory: IFluidDataStoreFactory,
     summaryVersion?: string,
     containerRuntimeFactoryType = ContainerRuntimeFactoryWithDefaultDataStore,
+    registryEntries?: NamedFluidDataStoreRegistryEntries,
```

Comment:
As per my comment in the call site, this is not needed, you can directly pass in the container runtime factory in the previous param.

Reply:
Are you sure ? In case we need to make use of ShaedMap, Counter, etc. I was getting errors without it.

Reply:
Yeah, you are right. That will not work.